release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -201,6 +201,19 @@ Previously, quick start prerequisites were displayed as combined text instead of
 
 image::quick-start-prerequisites.png[quick start prerequisites displayed as popover,338,336]
 
+[id="ocp-4-8-web-console-developer-perspective"]
+==== Developer perspective
+
+* As a cluster administrator, to create and deploy an application from a Git repository, you can use custom pipeline templates that override the default pipeline templates provided by {pipelines-title} 1.5 and later.
+* You can filter Helm charts based on their certification level and see all the Helm charts in the *Developer Catalog*.
+* The Add page now features new options to create applications and associated services and deploy them on {product-title}. These include *Getting started resources*, *Creating applications using samples*, *Build with guided documentation*, *Explore new developer features* to name a few. Users can optionally hide specific cards on the Add page.
+* You can use workspaces in pipelines when creating via the pipeline builder or adding a trigger to a pipeline so that tasks can be used to support workspaces in pipelines.
+* You can use the JAR files in the *Topology* view of the *Developer* perspective to deploy your Java applications.
+* You can create multiple event source types in {product-title} that can be connected to sinks. Functions deployed as a Knative service on a {product-title} cluster, can be connected as a sink to Knative Eventing components.
+* You can use the `finally` task in the pipeline to execute in parallel. The `finally` task can guaranteed to be executed in parallel after all the pipeline tasks have completed regardless of the status of the tasks.
+* You can use the code assistance in add task form for the task parameter values. The pipeline parameter and the correct syntax for referencing that particular pipeline parameter would be referenced in the respective text field.
```

Comment:
`Use the code assistance in the *Add Task* form to access the task parameter values. To view the pipeline parameter and the correct syntax for referencing that particular pipeline parameter, go to the respective text fields.`

Please confirm since I can't view the actual UI, but I made it **Add Task** form to be consistent with how you formatted **Topology** view and ** Developer** perspective. That is, you bold the names of UI elements, so I did that here for the **Add Tasks** form. Am I right that it should be capitalized? 

Also, am I right that you use the code assistance to *access* the parameter values?

Reply:
Yeah, it sounds good. Made the fix!